.circleci/config.yml

```diff
@@ -100,9 +104,14 @@ jobs:
       shell: bash --login -eo pipefail
     <<: *test-base
 
-  test-lambdas-indexer:
+  test-lambda:
     docker:
       - image: circleci/python:3.6-jessie
+    description: "Test lambdas"
+    parameters:
+      path:
+        description: "The lambda path"
```

Comment:
```suggestion
        description: "lambdas/$path"
```

Reply:
I don't think you can reference `parameters.path` inside the description of the parameter definition

Reply:
Oh, you mean just a literal string `"lambdas/$path"`